firebase-firestore/src/main/java/com/google/firebase/firestore/remote/Datastore.java

```diff
@@ -73,36 +71,17 @@
 
   private final FirestoreChannel channel;
 
-  private static Supplier<ManagedChannelBuilder<?>> overrideChannelBuilderSupplier;
```

Comment:
We can't get rid of this mechanism. It's used by the conformance tests in google3.

http://cs/search/?q=Datastore.overrideChannelBuilder

Reply:
Ah, so that's why it's there. Restored.